csun-metalab/nomi
# NAM-499 -- UI: Some of the buttons on desktop do not behave like links on mouse hover.

# Description
Added focus & hover color on clickable events.
  
# Testing
1. Hover through all buttons on NOMI.
2. Ensure an outline appears on all clickable events.
3. Make sure the test the colors on all themes for consistency.
